requirements.txt

```diff
@@ -41,7 +41,7 @@ orjson~=3.3
 alembic~=1.9
 mergedeep~=1.3
 # 3.2 iguazio system uses 0.9.x - limiting to only patch changes
```

Comment:
does the comment above is still relevant?

Reply:
Well, both parts of this statement are still true, but the comment as a whole is obsolete. I'll delete it.